src/EdpGithub/Api/AbstractApi.php

```diff
@@ -12,6 +12,11 @@
      */
     protected $client;
 
+    /**
+     * @var ServiceManager
+     */
+    private $serviceManager;
```

Comment:
The places where this property is accessed are the getters and setters - other than that, it's never accessed directly!